src/js/tabs/led_strip.js

```diff
@@ -396,6 +396,10 @@ led_strip.initialize = function (callback, scrollPosition) {
             },
         });
 
+        const ledStripSelectFunction = $('#ledStripFunctionSelect');
+        // Sort the element, if need to group, do it by lexical sort, ie. by naming of (the translated) selection text
+        ledStripSelectFunction.sortSelect(i18n.getMessage("ledStripFunctionNoneOption"));
+
```

Comment:
If we are only using the `ledStringSelectFunction` to sort it, better to do it in a single line.
```suggestion
        // Sort the element, if need to group, do it by lexical sort, ie. by naming of (the translated) selection text
        $('#ledStripFunctionSelect').sortSelect(i18n.getMessage("ledStripFunctionNoneOption"));

```
If we plan to use it for something more then is good to use the variable.

Reply:
Good suggestion, i'll change to that.